objects-and-methods/exercise-1/lib/trick_or_treater.rb

```diff
@@ -0,0 +1,34 @@
+class TrickOrTreater
+  attr_accessor :bag
+  def initialize(costume)
+    @bag = Bag.new
+    @costume = costume
+  end
+ 
+  def dressed_up_as
+    @costume.style
+  end
+  # def bag
+  #   @bag = Bag.new
+  # end
+
+  def has_candy?
+    if @bag == nil
+      return false 
+    else
+      !@bag.candies.empty?
+    end
+  end
+
+  def candy_count
+    if @bag == nil
+      return 0
+    else
+    @bag.candies.count
+    end
+  end
```

Comment:
Few improvements

1. Do alignment at 27

```suggestion
  def candy_count
    return 0 if @bag == nil

    @bag.candies.count
  end
```